### PR TITLE
Add babylon app registration

### PR DESCRIPTION
**Pull request by enocquet**
Babylon app registration added
To match other applications installed in this script, a sp and a password creation were also added
Tested on sample platform

### CODE REVIEW

**Review by vcarluer — CHANGES_REQUESTED** [review]
Usually we need to create Service Principal item also (azuread_service_principal in terraform), we create it for all other app registration

File: azure/create-platform-prerequisites/main.tf [hunk 1]
```diff
@@ -275,6 +275,45 @@ resource "azurerm_resource_group" "platform_rg" {
...
+  }
+
+  public_client {
+    redirect_uris = ["http://localhost:8484/"]
+  }
+}
+
+resource "azuread_service_principal" "babylon" {
+  depends_on                   = [azuread_service_principal.swagger]
+  application_id               = azuread_application.babylon[0].application_id
```
Comment (vcarluer) [inline azure/create-platform-prerequisites/main.tf:320]: I have commented by error on commit but to quickly say the same thing babylon[0] should be used only if there is a "Count=..." on the app registration. Add a var to make the app registration optional or change the application_id to azuread_application.babylon.application_id.
A "terraform validate" command may see this kind of issue.

**Review by vcarluer — CHANGES_REQUESTED** [review]
error in reference

File: azure/create-platform-prerequisites/main.tf (hunk 1 above)
Comment (enocquet) [reply azure/create-platform-prerequisites/main.tf:320]: changed

**Review by vcarluer — APPROVED** [review]
OK

**Pull request merged**